components/FAQ/index.js

```diff
@@ -0,0 +1,130 @@
+        `}
+      </style>
+
+      <Section.Header>
+        <Section.Title>FAQ</Section.Title>
+      </Section.Header>
+      <Section.Body>
+        <Container>
+          <Col
+            md={{ span: 6, offset: 3 }}
+            className={`text-center ${styles.faqPrompt}`}
+          >
+          </Col>
+          
+          <MediaQuery minWidth={767.98} >
+            <Row>
+            <Col xs="3">
+              <Button variant="navButton" onClick={() => setCurrSection('General')}>General</Button>
+              <Button variant="navButton" onClick={() => setCurrSection('Events')}>Events</Button>
+              <Button variant="navButton" onClick={() => setCurrSection('MechMania')}>MechMania</Button>
+              <Button variant="navButton" onClick={() => setCurrSection('PuzzleBang')}>PuzzleBang</Button>
+              <Button variant="navButton" onClick={() => setCurrSection('R|P Symposium for Presenters')}>R|P Symposium for Presenters</Button>
+              <Button variant="navButton" onClick={() => setCurrSection('R|P Symposium for Schools')}>R|P Symposium for Schools</Button>
+            </Col>
+            <Col>
+              <QuestionContainer
+                questions={currQuestions}
+                currQuestion={currQuestion}
+                handleToggle={handleToggle}
+              />
+            </Col>
+            </Row>
+          </MediaQuery>
+          <MediaQuery maxWidth={767.98} >
+              <div>
+              <Dropdown>
+                <Dropdown.Toggle
+                  size="lg"
+                  id="faq-dropdown"
+                  variant="secondary"
+                  className={styles.faqDropdown}
+                >
+                  {currSection}
+                </Dropdown.Toggle>
+                <Dropdown.Menu className={styles.faqDropdownMenu}>
+                  <Dropdown.Item className={styles.faqDropDownButton} onClick={() => setCurrSection('General')}>
+                    General
+                  </Dropdown.Item>
+                  <Dropdown.Item className={styles.faqDropDownButton} onClick={() => setCurrSection('Events')}>
+                    Events
+                  </Dropdown.Item>
+                  <Dropdown.Item className={styles.faqDropDownButton} onClick={() => setCurrSection('MechMania')}>
+                    MechMania
+                  </Dropdown.Item>
+                  <Dropdown.Item className={styles.faqDropDownButton} onClick={() => setCurrSection('PuzzleBang')}>
+                    PuzzleBang
+                  </Dropdown.Item>
+                  <Dropdown.Item className={styles.faqDropDownButton} onClick={() => setCurrSection('R|P Symposium for Presenters')}>
+                    R|P Symposium for Presenters
+                  </Dropdown.Item>
+                  <Dropdown.Item className={styles.faqDropDownButton} onClick={() => setCurrSection('R|P Symposium for Schools')}>
+                    R|P Symposium for Schools
+                  </Dropdown.Item>
```

Comment:
Can this mapped instead of repeatiinng the same code 6 times...something like
```suggestion
     {FAQSections.map((section) => (
                  <Dropdown.Item className={styles.faqDropDownButton} onClick={() => setCurrSection(section)}>
                      {section}
                  </Dropdown.Item>
      ))}
```

Reply:
Not sure what's going on with the mobile view on your end. Here's what it looks like for me:
![image](https://user-images.githubusercontent.com/70115186/130267534-a06482eb-fc54-45d2-bf82-b9741262d03b.png).

I also tried some things to get rid of the repeated code you highlighted, but it creates a runtime error when interacting with the buttons. We can discuss the specific error & try to resolve it in a call if you'd like.